docs/Catalog/FileBrowser.md

```diff
@@ -41,3 +41,33 @@ format (the default is README.md), enter your content, and click save.
 ![Choose name](../imgs/catalog-texteditor-name.png)
 
 ![Edit file](../imgs/catalog-texteditor-main.png)
+
+## Working with Amazon S3 Glacier storage classes
+
+Glacier storage classes are built for data archiving. Quilt is
+compatible with S3 bucket lifecycle rules that transition S3 objects to
+Glacier storage classes.
+
+There are several types of S3 object archive storage class that
+working differently with the Quilt Catalog (and API).
```

Comment:
Also need to mention, maybe moving the page to just have a page on Glacier, that this isn't catalog specific and these same issues will affect e.g. `quilt3 install`.